docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -52,6 +52,8 @@ Access
 - Access to Amazon `S3`_ using `gsutil`_ on your :doc:`Flocker development machine <vagrant>`.
   Set ``aws_access_key_id`` and ``aws_secret_access_key`` in the ``[Credentials]`` section of ``~/.boto``.
 
+- Access to `S3` and CloudFront via web.
```

Comment:
We don't have this for other prerequisites but something we found with the latest weekly release was that it is unclear what is a ClusterHQ shared account and what is a personal account with permissions. If this could be clarified here that will be useful.